src/components/call-tool.js

```diff
@@ -130,7 +131,15 @@ module.exports = React.createClass({
       <section>
         <h2 className="bold">{this.context.intl.formatMessage({id: 'calling_headline'})}</h2>
         <div>{this.context.intl.formatMessage({id: 'calling_tagline'})}</div>
-        <div className="bold">SOCIAL ICONS GO HERE</div>
+        <Social/>
+        <p className="try-again">
+          <FormattedMessage
+            id='call_again'
+            values={{
+              callAgain: (<a href="#">{this.context.intl.formatMessage({id: 'call_again_link'})}</a>)
```

Comment:
switch this to `href={ `/${this.context.intl.locale}/call-now }` and we should be good.